### PR TITLE
Fix invalid output for template string

### LINKED ISSUE #100: Invalid output for template string
Test:

```js
var cst = require('cst');
var Parser = cst.Parser;

var parser = new Parser();
var program = parser.parse('`${1}`');
console.log(program.sourceCode);
```

**Output**

```
`template${1}template`
```

**Expected output**

```js
`${1}`
```

### PR DESCRIPTION
**Pull request by ikokostya**
Fixes #100

### CODE REVIEW

File: src/elementTree.js [hunk 1]
```diff
@@ -217,9 +217,7 @@ function processToken(token: Object, source: string): BabylonToken {
         }
     } else if (type === tt.template) {
         token.type = 'Template';
-        if (!token.value) {
-            token.sourceCode = token.value = type.label;
-        }
+        token.sourceCode = token.value;
```
Comment (zxqfox) [inline src/elementTree.js]: Should it be `if (token.hasOwnProperty('value'))` instead of `!token.value`?

**Comment by mdevils** [comment]
Good job!

**Comment by mdevils** [comment]
Fixed in `cst@0.0.28`

**Comment by hzoo** [comment]
Can you push tags? The latest is 0.0.26 on github

**Comment by mdevils** [comment]
Done :)

**Pull request merged**